packages/network-controller/src/create-network-client.ts

```diff
@@ -49,11 +48,6 @@ export function createNetworkClient(
 
   const provider = createMetamaskProvider(providerConfig);
 
-  assert(
```

Comment:
ProviderEngine always sets a block tracker property in its constructor, so there is no need to assert for it.